spark/interpreter/src/main/java/org/apache/zeppelin/spark/PySparkInterpreter.java

```diff
@@ -204,33 +204,35 @@ private Map setupPySparkEnv() throws IOException, InterpreterException {
 
     // only set PYTHONPATH in local or yarn-client mode.
     // yarn-cluster will setup PYTHONPATH automatically.
-    SparkConf conf = getSparkConf();
+    SparkConf conf = null;
+    try {
+      conf = getSparkConf();
```

Comment:
what's the reason that getSparkConf can fail here?

Reply:
I'm just copied from #3374. 
I can simplify if necessary.
@zjffdu